src/renderers/webgl/WebGLIndexedBufferRenderer.js

```diff
@@ -14,24 +14,10 @@ function WebGLIndexedBufferRenderer( gl, extensions, infoRender ) {
 
 	var type, size;
 
-	function setIndex( index ) {
+	function setIndex( indexProperties ) {
```

Comment:
It's only called in one place as far as I know, so two functions seems like overkill.

setProperties()  makes me wonder: set properties of what?
setIndexProperties() seems more logical to me?